ProcessLib/HydroMechanics/HydroMechanicsProcess.cpp

```diff
@@ -295,6 +295,11 @@ void HydroMechanicsProcess<DisplacementDim>::initializeConcreteProcess(
             const_cast<MeshLib::Mesh&>(mesh), "principal_stress_values",
             MeshLib::MeshItemType::Cell, 3);
 
+    _process_data.permeability = MeshLib::getOrCreateMeshProperty<double>(
```

Comment:
It would be nice  make it as an optional  initialization to save memory for  homogeneous or piece wise homogeneous material.

Reply:
Just a suggestion. The other parts looks good.

Reply:
I added a commit to make the output optional